contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -2920,6 +2923,31 @@ sp_rename_internal(PG_FUNCTION_ARGS)
 				curr_relname[--len] = 0;
 		}
 
+		/* remove delimited identifiers if quoted_identifier is on */
+		if (pltsql_quoted_identifier)
+		{
+			len = strlen(obj_name);
+			if ((obj_name[0] == '"' && obj_name[len - 1] == '"') || (obj_name[0] == '[' && obj_name[len - 1] == ']'))
+			{
+				memmove(obj_name, &obj_name[1], len - 1);
+    			obj_name[len - 2] = '\0';
+			}
```

Comment:
Please create a helper function for this so you can call it for a generic char array so it's much cleaner.

foo(obj_name);
foo(schema_name);
foo(curr_relname);

Reply:
Also, curious, will the string have any leading and trailing space?